pkg/artifactory/provider.go

```diff
@@ -7,9 +7,8 @@ import (
 
 	artifactoryold "github.com/atlassian/go-artifactory/v2/artifactory"
 	"github.com/atlassian/go-artifactory/v2/artifactory/transport"
-	"github.com/hashicorp/terraform/helper/schema"
-	"github.com/hashicorp/terraform/terraform"
-	"github.com/hashicorp/terraform/version"
```

Comment:
This is deprecated and required some code changes to update. These code changes are standard and recommended by `tf-sdk-migrator`